ofl/mochiypopone/METADATA.pb

```diff
@@ -0,0 +1,19 @@
+name: "Mochiy Pop One"
+designer: "FONTDASU"
+license: "OFL"
+category: "HANDWRITING"
+date_added: "2020-12-07"
+fonts {
+  name: "Mochiy Pop One"
+  style: "normal"
+  weight: 400
+  filename: "MochiyPopOne-Regular.ttf"
+  post_script_name: "MochiyPopOne-Regular"
+  full_name: "Mochiy Pop One Regular"
+  copyright: "Copyright 2020 The MochiyPop Project Authors (https://github.com/fontdasu/Mochiypop)"
+}
+subsets: "chinese-simplified"
```

Comment:
There's no chinese so remove that

Reply:
From what I can tell, there's no way to create the METADATA.pb file locally and upload to the server via the gftools packager. Is the only way to edit this after I run packager?